frontend/public/components/chargeback.tsx

```diff
@@ -51,26 +52,26 @@ const dataURL = (obj, format='json') => {
   const serviceModel = modelFor('Service');
   return resourceURL(serviceModel, {
     ns: obj.metadata.namespace,
-    name: 'chargeback',
+    name: 'https:reporting-operator:http',
     path: 'proxy/api/v1/reports/get',
     queryParams: {
       name: obj.metadata.name,
       format,
+      namespace: obj.metadata.namespace,
     },
   });
 };
 
 const ChargebackNavBar: React.SFC<{match: {url: string}}> = props => <div>
   <PageHeading title="Chargeback Reporting" style={{paddingBottom: 15}} />
-  <NavBar pages={reportPages} basePath={props.match.url.split('/').slice(0, -1).join('/')} />
+  <NavBar pages={reportPages} basePath={props.match.url.split('/').slice(0, -1).join('/')} hideDivider />
 </div>;
 
 
 const ReportsHeader = props => <ListHeader>
   <ColHead {...props} className="col-lg-3 col-md-3 col-xs-4" sortField="metadata.name">Name</ColHead>
   <ColHead {...props} className="col-lg-2 col-md-3 col-xs-4" sortField="metadata.namespace">Namespace</ColHead>
   <ColHead {...props} className="col-lg-2 hidden-md hidden-sm hidden-xs">Report Generation Query</ColHead>
-  <ColHead {...props} className="col-lg-1 col-md-2 col-xs-4" sortField="spec.status.phase">Status</ColHead>
```

Comment:
We should adjust the column widths so they still total 12

Reply:
Fixed